elide-standalone/src/main/java/com/yahoo/elide/standalone/Util.java

```diff
@@ -89,4 +135,39 @@ public static EntityManagerFactory getEntityManagerFactory(String modelPackageNa
                 .map(Class::getName)
                 .collect(Collectors.toList());
     }
+
+    /**
+     * Find classes with a particular annotation from dynamic compiler.
+     * @param compiler An instance of ElideDynamicEntityCompiler.
+     * @param annotationClass Annotation to search for.
+     * @return Set of Classes matching the annotation.
+     * @throws ClassNotFoundException ClassNameNotFound in compiler
+     */
+    @SuppressWarnings({ "unchecked", "rawtypes" })
+    public static List<String> findAnnotatedClassNames(ElideDynamicEntityCompiler compiler, Class annotationClass)
```

Comment:
Maybe this can be a non-static method on the compiler (to return the list of classes that it manages).